### ISSUE TITLE
Bear import skip file attachements with same name

### ISSUE THREAD

**Issue by jmrauen** [obsidianmd/obsidian-importer#67]
In bear each textbundle has an own folder for images. But in the import each file will be put in the same assets-folder. 
So when I have more notes with an image e.g. "untitled.jpg", only one image of them will appear in each note!

Please rename an existing filename e.g. untitled123.jpg and not skip the import. Thanks:)

**Comment by arthurtyukayev** [comment]
@lishid hoping for your input on this, maybe I can just update the importer to nest the assets for each markdown file in a folder in `Bear/assets`?

**Comment by eware-godaddy** [comment]
Seeing this same issue. I know have hundreds of notes linking to the wrong images. What's the function that finds a unique available name?

**Comment by nicolashohm** [comment]
The importer anyways [modifies the asset path in the markdown.](https://github.com/obsidianmd/obsidian-importer/blob/master/src/formats/bear-bear2bk.ts#L47) Some kind of unique identifier (like a hash of the note's file name) referring to the note could be added here.

**Comment by torb-no** [comment]
I noticed a lot of my images where the same one. Frequently this was the first import with a file named ”image.png” which Bear seems to use a lot.

If you export Bear to markdown (instead of using the backup format) you get a lot of these:

- /some-note-1/image.png
- /a-different-note-2/image.png
- etc

I can imagine there being something similar stored in the Bear backup file and it seems like the importer is completely failing in dealing with this. I wouldn't be surprised if this is connected the issue with skipping all the files as well (maybe the skipped files are all the ”image.png” files).

I think it's pretty serious that an official importer plugin as such profound data loss. The main reason I discovered is because I'm pretty paranoid about the reliability of Obsidian from previous experiences so I doubled checked, but I reckon less technically inclined people might just follow official Obsidian documentation and get severely burned.
